server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,199 @@
+req_source_count_df = req_source_count_df[:5]
+
+req_source_bar_chart = px.bar(
+    req_source_count_df,
+    y=req_source_count_df.index,
+    x='counts',
+    title="Total Requests by Source",
+    orientation='h'
+)
+
+# Median Request Days to Close Box Plot
+REQ_TYPE_STATS_API_PATH  = '/types/stats'
+print(" * Downloading data from API path: " + REQ_TYPE_STATS_API_PATH)
+stats_df = pd.read_json(API_HOST + REQ_TYPE_STATS_API_PATH)
+stats_df = stats_df.sort_values('median', ascending=False)
+print(" * Dataframe has been loaded from API path: " + REQ_TYPE_STATS_API_PATH)
+
+med_days_to_close_box_plot = go.Figure()
+med_days_to_close_box_plot.add_trace(
+    go.Box(
+        y=stats_df.type_name,
+        q1=stats_df['q1'],
+        median=stats_df['median'],
+        q3=stats_df['q3'],
+        marker_color='#29404F',
+        fillcolor='#E17C05',
+    )
+)
+med_days_to_close_box_plot.update_xaxes(
+    dtick=5
+)
+med_days_to_close_box_plot.update_layout(
+    title="Total Median Days to Close by Type",
+)
+
+# Day of Week Bar Chart:
+start_date = datetime.date.today() - datetime.timedelta(days=365)
+end_date = datetime.date.today() - datetime.timedelta(days=1)
+date_range_req_data_params = {'filter':f"created_date>={start_date}", 'filter':f"created_date<={end_date}"}
+DATE_RANGE_REQ_DATA_API_PATH = REPORT_API_PATH_ROOT + urllib.parse.urlencode(date_range_req_data_params)
+print(" * Downloading data for dataframe from API path: " + DATE_RANGE_REQ_DATA_API_PATH)
+date_range_req_df = pd.read_json(API_HOST + DATE_RANGE_REQ_DATA_API_PATH)
+print(" * Dataframe has been loaded from API path: " + DATE_RANGE_REQ_DATA_API_PATH)
+date_range_req_df['created_date'] = pd.to_datetime(date_range_req_df['created_date'])
+day_of_week_df = date_range_req_df.groupby(['created_date']).agg('sum').reset_index()
+day_of_week_df['day_of_week'] = day_of_week_df['created_date'].dt.day_name()
+
+num_req_by_day_bar_chart = px.bar(
+    day_of_week_df,
+    x="day_of_week",
+    y="counts",
+    labels=LABELS,
+)
+num_req_by_day_bar_chart.update_xaxes(categoryorder='array', categoryarray=[
+    "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"])
+
+# Total Request by NC
+req_by_nc_bar_chart = px.bar(
+    nc_req_count_df,
+    x=nc_req_count_df.index,
+    y='counts',
+    labels=LABELS,
+    title="Total Requests by Neighborhood Councils",
+)
+req_by_nc_bar_chart.update_layout(font=dict(size=12))
+
+# LAYOUT VARIABLES
+INDICATOR_CARD_STYLE = {"text-align": 'center',
+    "border": "0.5px black solid", 'width': '18vw', 'display': 'inline-block'}
+BORDER_STYLE = {"border": "0.5px black solid"}
+EQUAL_SPACE_BOX_STYLE = {'display': 'flex', "justify-content": "space-between"}
+TWO_CHART_STYLE = {"width": "48vw", "height": "40vh"}
+DASHBOARD_OUTLINE = "The figures below represent the total number of 311 requests made across LA County from 2016-2021. In 2020, we saw an all-time high with more than 1.4 million requests."
```

Comment:
Let's break this into multiple lines so that it's easier to read.

Reply:
Thanks for the suggestion - resolved.